elements/pfe-sass/maps/_general.scss

```diff
@@ -26,9 +26,11 @@ $pfe-vars: (
   surface--border-width--heavy:  4px,
   surface--border-style:         solid,
   surface--border-radius:        3px,
+  surface--border-width--active: 3px,
+  surface--border-width--color:  #151515,
```

Comment:
These exist in the color map as:

```
// Palette of Border Colors
  surface--border--lightest:    $pfe-color--gray-100,
  surface--border--lighter:      $pfe-color--gray-100,	
  surface--border:                    $pfelements--gray-light,
  surface--border--darker:      $pfe-color--gray-300,
  surface--border--darkest:    $pfe-color--gray-800,
```

Reply:
@starryeyez024 This one I would label a #rock and should probably be tidied so we don't end up with disparate color properties.

Reply:
@starryeyez024 bump

Reply:
not sure what surface--border-width--color means (width or color)? but it was unused so I removed it